### ISSUE TITLE
[axe-core warning] - Need to ensure img elements have valid alternate text or a role of none or presentation.

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#653]
# Feature Request

Need to ensure img elements have valid alternate text or a role of none or presentation.

## Description
<details>
 <summary>WDIO Axe Error Report</summary>

```
ApplicationLayout Navigates from primary nav menu when small is accessible:
expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"https://github.com/cerner/terra-framework/raw/master/terra.png\" alt=\" \" height=\"26px\" width=\"26px\" class=\"Image-module__image___yfp6a Image-module__rounded___1yRN_\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        ".ApplicationHeaderName-module__accessory___3Z_MP > img"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
running chrome
AssertionError: expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"https://github.com/cerner/terra-framework/raw/master/terra.png\" alt=\" \" height=\"26px\" width=\"26px\" class=\"Image-module__image___yfp6a Image-module__rounded___1yRN_\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        ".ApplicationHeaderName-module__accessory___3Z_MP > img"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
    at Context.<anonymous> (/Users/bj031910/repos/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessiblity.js:13:54)
    at new Promise (<anonymous>)
    at new F (/Users/bj031910/repos/terra-framework/node_modules/core-js/library/modules/_export.js:36:28)

```
</details>

<details>
 <summary>WDIO Axe Error Report</summary>

```
ApplicationLayout Presents utility menu from header when large is accessible:
expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"https://github.com/cerner/terra-framework/raw/master/terra.png\" alt=\" \" height=\"26px\" width=\"26px\" class=\"Image-module__image___yfp6a Image-module__rounded___1yRN_\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        "img"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
running chrome
AssertionError: expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"https://github.com/cerner/terra-framework/raw/master/terra.png\" alt=\" \" height=\"26px\" width=\"26px\" class=\"Image-module__image___yfp6a Image-module__rounded___1yRN_\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        "img"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
    at Context.<anonymous> (/Users/bj031910/repos/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessiblity.js:13:54)
    at new Promise (<anonymous>)
    at new F (/Users/bj031910/repos/terra-framework/node_modules/core-js/library/modules/_export.js:36:28)

```
</details>

<details>
 <summary>WDIO Axe Error Report</summary>

```
ApplicationLayout Presents utility menu from header and checks for closure on read-only item click is accessible:
expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"https://github.com/cerner/terra-framework/raw/master/terra.png\" alt=\" \" height=\"26px\" width=\"26px\" class=\"Image-module__image___yfp6a Image-module__rounded___1yRN_\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        "img"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
running chrome
AssertionError: expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"https://github.com/cerner/terra-framework/raw/master/terra.png\" alt=\" \" height=\"26px\" width=\"26px\" class=\"Image-module__image___yfp6a Image-module__rounded___1yRN_\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        "img"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
    at Context.<anonymous> (/Users/bj031910/repos/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessiblity.js:13:54)
    at new Promise (<anonymous>)
    at new F (/Users/bj031910/repos/terra-framework/node_modules/core-js/library/modules/_export.js:36:28)
```
</details>

**Comment by bjankord** [comment]
This has been resolved in a prior PR and is no longer being reported when running axe-core with the WCAG 2.1 AA ruleset.
